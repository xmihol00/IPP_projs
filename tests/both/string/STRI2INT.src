.IPPcode21

DEFVAR GF@ord
DEFVAR GF@str
DEFVAR GF@pos

MOVE GF@str string@ABCDEFG
MOVE GF@pos int@5
STRI2INT GF@ord GF@str GF@pos
WRITE GF@ord
WRITE string@\010

STRI2INT GF@ord GF@str int@0
WRITE GF@ord
WRITE string@\010

STRI2INT GF@ord GF@str int@6
WRITE GF@ord
WRITE string@\010

STRI2INT GF@ord string@abd int@1
WRITE GF@ord
WRITE string@\010

STRI2INT GF@ord string@aabbcc GF@pos
WRITE GF@ord
WRITE string@\010
