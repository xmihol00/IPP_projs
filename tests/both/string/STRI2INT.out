70
65
71
98
99
